apps/wing/src/cli.ts

```diff
@@ -100,6 +102,27 @@ async function main() {
     }
   }
 
+  async function watchHook(cmd: Command) {
+    const entrypoint = cmd.args[0];
+    const watch = cmd.opts().watch;
+    if (watch) {
+      const watcher = chokidar.watch(entrypoint, {
+        ignoreInitial: true,
+      });
+
+      log(`Starting to watch ${entrypoint}`);
+
+      watcher.on("change", async () => {
+        log(`Wing source file changed, rerunning tests…`);
+        await import(`./commands/test`).then((m) => m["test"](cmd.args, cmd.opts()));
```

Comment:
I think we need to make sure the top level options are carried through, just in case (might prevent subtle bugs later)

```suggestion
        await import(`./commands/test`).then((m) => m["test"](cmd.args, cmd.optsWithGlobals()));
```